README.md

```diff
@@ -2,19 +2,25 @@
 This repo contains a fixture to enable testing REST services with JSON, XML or Text payloads
 
 # Getting Started
-1. Download FitNesse (http://fitnesse.org) and install it to C:\Apps\FitNesse
-2. Download FitSharp (https://github.com/jediwhale/fitsharp) and install it to C:\Apps\FitNesse\FitSharp.
-3. Clone the repo to a local folder (C:\Data\FitNesseDemo)
-4. Update plugins.properties to point to the FitSharp folder (if you took other folders than suggested)
-5. Build all projects in the solution Rest (Release)
-6. Ensure you have Java installed (1.7 or higher)
-7. Start FitNesse with the root repo folder as the data folder as well as the current directory:
+1. Download FitNesse (http://fitnesse.org) and install it to <I>C:\Apps\FitNesse</I>
+2. Download FitSharp (https://github.com/jediwhale/fitsharp) and install it to <I>C:\Apps\FitNesse\FitSharp</I>.
+   <br />Or install the package via nuget: 
+   ```
+   nuget install FitSharp
```

Comment:
If you do it via NuGet, you either need to copy the net5.0 DLLs to C:\Apps\FitSharp\2.8.2.1\net5.0 or change the FITSHARP_HOME variable in plugins.properties to where NuGet puts those DLLs ${USERPROFILE}\\.nuget\\packages\\fitsharp\\2.8.2.1\\lib\\net5.0

Reply:
in Nuget install, you can specify the output folder with switch: -OutputDirectory (https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-install). e.g. nuget install FitSharp -OutputDirectory C:\Apps\FitNesse